lib/gollum-lib/filter/render.rb

```diff
@@ -3,7 +3,11 @@
 class Gollum::Filter::Render < Gollum::Filter
   def extract(data)
     begin
-      data = GitHub::Markup.render(@markup.name, data)
+      working_dir = ::File.join(@markup.wiki.path, @markup.dir)
```

Comment:
For consistency, perhaps a `pathname` solution would be better here?

Reply:
👍